### PR TITLE
Update hexbin and dsra source

### PR DESCRIPTION
**Pull request by drotheram**
Change source repository for DSRA models
Added support for Hexbin geometries PostGIS and ElasticSearch 

Update some python ES loading scripts to take advantage of modularized refactor
Common ES index prefix to distinguish from system indices
Use of Kibana Spaces

closes 98, 103, 119, 120

### CODE REVIEW

**Review by DamonU2 — APPROVED** [review]

**Review by wkhchow — APPROVED** [review]
looks good to me

File: python/socialFabric_postgres2es.py [hunk 1]
```diff
@@ -1,205 +1,120 @@
...
-                    },
-                    'geometry': {
-                        'type': 'geo_shape'
+            }),
+            view="opendrr_nhsl_social_fabric_all_indicators_{agg}".format(**{
+                'agg': aggregation}),
+            sqlquerystring='SELECT *, ST_AsGeoJSON({geom}) \
+                FROM \
+                results_nhsl_social_fabric.nhsl_social_fabric_all_indicators_{agg} \
+                ORDER BY "{sort_field}" \
```
Comment (jvanulde) [inline python/socialFabric_postgres2es.py:57]: Just out of curiosity, why are we sorting?

**Review by jvanulde — APPROVED** [review]
Great work! This looks much cleaner. Perhaps we can think of ways to refactor further such that we don't have a separate script for each *_postgres2es.py.

File: python/socialFabric_postgres2es.py (hunk 1 above)
Comment (drotheram) [reply python/socialFabric_postgres2es.py:57]: This was on @wkhchow's recommendation to make sure we get the same order of records when we apply LIMIT/OFFSET pagination

**Pull request merged**